pyo3-ffi/src/compat/py_3_13.rs

```diff
@@ -37,3 +37,36 @@ compat_function!(
         item
     }
 );
+
+compat_function!(
+    originally_defined_for(Py_3_13);
+
+    #[inline]
+    pub unsafe fn PyWeakref_GetRef(
+        _ref: *mut crate::PyObject,
```

Comment:
```suggestion
        ref: *mut crate::PyObject,
```

same for all the other uses of `_ref`

Reply:
I think `ref` gets rejected as a keyword. Maybe `ref_` is slightly better? Or `r#ref`. What do you prefer @mejrs?

(I think typically we've gone with trailing underscore e.g. `abstract_`.)

Reply:
Oh, right. I think `ref_` is best then

Reply:
Or just `reference`.